requirements.txt

```diff
@@ -0,0 +1,4 @@
+flake8
+pytest
```

Comment:
My configuration was copied from here https://github.com/signalfx/collectd-rabbitmq. It looks like both projects have the save issue. 
Any way I'll fix it in next commit.